projects/aca-content/assets/app.extensions.json

```diff
@@ -1532,7 +1532,27 @@
               "visible": "app.areCategoriesEnabled"
             }
           }
-        ]
+        ],
+        "highlight": {
+          "prefix": "<span class='aca-highlight'>",
```

Comment:
maybe worth having some sort of `template` property that includes both pre- and post-? i.e.

```json
{
  "template": "<span>{{value}}</span>"
}
```

where you substitute `{{value}}` with the highlighted term

Reply:
How would that work? BE expects us to provide prefix and postfix separately to properly highlight the search term, and what then is passed inside prefix and postfix is decided by search API

Reply:
ah, ok, no worries then, I thought it's combined in the FE